app/models/user.rb

```diff
@@ -8,5 +8,5 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
 
-  # attr_reader :verification_code
+  has_many :rents, dependent: :destroy
```

Comment:
Are you sure to want to use `:destroy` option here? (Open question, this will not imply a change but an argument response)

Reply:
I thought it made sense in the context of the application.
A rent links a user and a book together. If, for some reason, a user is deleted I assumed his rents should be deleted as well as there would no user to link to the book anymore.

Reply:
Ok, and if the owner of the page would like to have an history of all rents on a book, just to know how many times a book was rents, independently of who rent it? ((Open question, this will not imply a change but an argument response)

Reply:
That's a good point. I guess they could keep a counter on the Book that is increased by 1 every time a Rent is created if they want to know how many times it was rented. That way even if a Rent is deleted, the Book can still keep track of how many times it was rented.

Reply:
Good point 👍 
You can also use `dependent :nullify` and set user_id as not required and keep the rents ;)